Another error with cuda 7.5

@psychocrypt I also get errors while building (dry system without a card)

```
fireice@LAP440:~/xmr-stak-nvidia$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2015 NVIDIA Corporation
Built on Tue_Aug_11_14:27:32_CDT_2015
Cuda compilation tools, release 7.5, V7.5.17
fireice@LAP440:~/xmr-stak-nvidia$ gcc --version
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.4) 5.4.0 20160609
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
fireice@LAP440:~/xmr-stak-nvidia/build$ cmake -DCUDA_ARCH=50 ..
-- Configuring done
-- Generating done
-- Build files have been written to: /home/fireice/xmr-stak-nvidia/build
fireice@LAP440:~/xmr-stak-nvidia/build$ make
[  4%] Building NVCC (Device) object CMakeFiles/xmr-stak-nvidiaCuda.dir/nvcc_code/xmr-stak-nvidiaCuda_generated_cuda_core.cu.o
/usr/include/string.h: In function ‘void* __mempcpy_inline(void*, const void*, size_t)’:
/usr/include/string.h:652:42: error: ‘memcpy’ was not declared in this scope
   return (char *) memcpy (__dest, __src, __n) + __n;
                                          ^
CMake Error at xmr-stak-nvidiaCuda_generated_cuda_core.cu.o.cmake:266 (message):
  Error generating file
  /home/fireice/xmr-stak-nvidia/build/CMakeFiles/xmr-stak-nvidiaCuda.dir/nvcc_code/./xmr-stak-nvidiaCuda_generated_cuda_core.cu.o


CMakeFiles/xmr-stak-nvidiaCuda.dir/build.make:205: recipe for target 'CMakeFiles/xmr-stak-nvidiaCuda.dir/nvcc_code/xmr-stak-nvidiaCuda_generated_cuda_core.cu.o' failed
make[2]: *** [CMakeFiles/xmr-stak-nvidiaCuda.dir/nvcc_code/xmr-stak-nvidiaCuda_generated_cuda_core.cu.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/xmr-stak-nvidiaCuda.dir/all' failed
make[1]: *** [CMakeFiles/xmr-stak-nvidiaCuda.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -103,7 +103,10 @@ if("${CUDA_COMPILER}" STREQUAL "clang")
     endforeach()
 
 elseif("${CUDA_COMPILER}" STREQUAL "nvcc")
-
+    # avoid that nvcc in CUDA < 8 tries to use libc `memcpy` within the kernel
+    if(CUDA_VERSION VERSION_LESS 8.0)
+        add_definitions(-D_FORCE_INLINES)
+    endif()
     foreach(CUDA_ARCH_ELEM ${CUDA_ARCH})
         # set flags to create device code for the given architecture
         set(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS}

--- a/nvcc_code/cuda_extra.cu
+++ b/nvcc_code/cuda_extra.cu
@@ -3,6 +3,7 @@
 #include <string.h>
 #include <cuda.h>
 #include <cuda_runtime.h>
+#include <device_functions.hpp>
 
 #ifdef __CUDACC__
 __constant__

